tasks/prepare-exodus-params/README.md

```diff
@@ -1,14 +1,19 @@
 # prepare-exodus-params
 
 Tekton task that prepares exodus configuration options from the data file. The task looks at the data file
-in the workspace to extract the `exodus.env` key. Depending on the values, correct configuration options are returned. 
+in the workspace to extract the `cdn.env` key. Depending on the values, correct configuration options are returned. 
+
+exodus-gw is a microservice dedicated to writing data onto the CDN.
 
 ## Parameters
 
 | Name     | Description                                                                                                                 | Optional | Default value |
 |----------|-----------------------------------------------------------------------------------------------------------------------------|----------|---------------|
 | dataPath | Path to the merged data JSON file generated by collect-data task and containing the exodus configuration options to use     | No       | -             |
 
+## Changes in 0.2.2
+* Use `cdn.env` instead of `exodus.env` to align RPA data with other pipelines
```

Comment:
Whatever *is* read from the exodus key should be in https://github.com/konflux-ci/release-service-catalog/blob/development/schema/dataKeys.json

Reply:
cdn is already part of the schema here! so this change also aligns us with that! 

https://github.com/konflux-ci/release-service-catalog/blob/development/schema/dataKeys.json#L449-L456

Reply:
Yeah, I am saying if the exodus key, not cdn key, is still relevant for anything, it should be in the schema (if this change removes exodus entirely, nothing to do)